features/api_package_create.feature

```diff
@@ -14,6 +14,10 @@ Feature: Create a package from the API
       | sample1.txt | 01/12/2011 13:45 | sean@intersect.org.au  | 1/6/2010 6:42:01 | 30/11/2011 18:05:23 | samples/sample1.txt | 1  |
       | sample2.txt | 30/11/2011 10:15 | admin@intersect.org.au | 1/6/2010 6:42:01 | 30/11/2011 18:05:23 | samples/sample2.txt | 2  |
       | sample3.txt | 30/12/2011 12:34 | admin@intersect.org.au |                  |                     | samples/sample3.txt | 3  |
+      | error_file.txt | 30/12/2011 12:34 | admin@intersect.org.au |                  |                     | samples/error_file.txt | 4  |
+      | incomplete_package.zip | 30/12/2011 12:34 | admin@intersect.org.au |                  |                     | samples/incomplete_package.zip | 5  |
+    And I set file "error_file.txt" as error file
```

Comment:
You should change this method to make it more generic... Something like:
And I set the file processing status to "ERROR" for file "error_file.txt"
This way, we can use the same function to set any file processing status....